### PR TITLE
Closes #948 Update x.SUBJECT/S and x.overview

### LINKED ISSUE #948: Update x.SUBJECT/S and x.overview for subjects with illegal characters
### Description

- Found this after some more extensive testing of the `EMI_001` dataset 
- We need to update `x.SUBJECT/S` and `x.overview` for subjects with illegal characters after DCM2NII
- We didn't see this bug before, because we only ran each import sub-module individually, the failure only occurs if you run steps one and two in one go though

### Release notes

Update x.SUBJECT/S and x.overview for subjects with illegal characters.

### PR DESCRIPTION
**Pull request by MichaelStritt**
### Linked issue

Check out the description in #948

### Comments

Seems like we overlooked this minor bug in the previous merge.

### CODE REVIEW

File: Modules/SubModule_Import/xASL_imp_DCM2NII.m [hunk 1]
```diff
@@ -1,4 +1,4 @@
-function xASL_imp_DCM2NII(x, imPar)
+function x = xASL_imp_DCM2NII(x, imPar)
 %xASL_imp_DCM2NII Run the dcm2nii part of the import.
 %
 % FORMAT: xASL_imp_DCM2NII(x, imPar)
```
Comment (jan-petr) [inline Modules/SubModule_Import/xASL_imp_DCM2NII.m:4]: "x = " missing

**Review by jan-petr — APPROVED** [review]
OK.

**Review by jan-petr — CHANGES_REQUESTED** [review]
One minor header thing

File: Modules/SubModule_Import/xASL_imp_DCM2NII.m (hunk 1 above)
Comment (MichaelStritt) [reply Modules/SubModule_Import/xASL_imp_DCM2NII.m]: Fixed

**Review by jan-petr — APPROVED** [review]
OK

**Pull request merged**